drivers/video/video_ctrls.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_VIDEO_CTRLS_H_
+#define ZEPHYR_INCLUDE_VIDEO_CTRLS_H_
+
+#include <zephyr/device.h>
+#include <zephyr/sys/slist.h>
+
+/*  Control flags */
+#define VIDEO_CTRL_FLAG_READ_ONLY        0x0004
+#define VIDEO_CTRL_FLAG_UPDATE           0x0008
+#define VIDEO_CTRL_FLAG_INACTIVE         0x0010
+#define VIDEO_CTRL_FLAG_WRITE_ONLY       0x0040
+#define VIDEO_CTRL_FLAG_VOLATILE         0x0080
+#define VIDEO_CTRL_FLAG_EXECUTE_ON_WRITE 0x0200
```

Comment:
Did Linux compatibility of the internal encoding of the control flags helped with something in particular?
Zephyr does not have an ABI to maintain insofar AFAIK.

```suggestion
#define VIDEO_CTRL_FLAG_READ_ONLY        BIT(0)
#define VIDEO_CTRL_FLAG_UPDATE           BIT(1)
#define VIDEO_CTRL_FLAG_INACTIVE         BIT(2)
#define VIDEO_CTRL_FLAG_WRITE_ONLY       BIT(3)
#define VIDEO_CTRL_FLAG_VOLATILE         BIT(4)
#define VIDEO_CTRL_FLAG_EXECUTE_ON_WRITE BIT(5)
```

There also seems a few items not used throught the code, I assume part of a few TODO left...

Reply:
Yes, I think we do not need to keep compatibility with Linux as this is internal stuffs. That makes thing easier.